examples/secure-roks-cluster/testing.auto.tfvars

```diff
@@ -0,0 +1,10 @@
+##############################################################################
+## Global Variables
+##############################################################################
+
+# ibmcloud_api_key = ""
+region                    = "eu-de" # eu-de for Frankfurt MZR
+resource_group            = "secure"
+resource_prefix           = "secure"
+number_of_addresses       = 256
+activity_tracker_instance = "5129f485-0195-4184-abe5-84716cb80e68"
```

Comment:
probably want to remove this id.

Reply:
I commented out this activity_tracker_instance.